config/development.yaml

```diff
@@ -82,19 +82,27 @@ dcRedirectionPolicy:
   toDC: ""
 
 archival:
-  status: "enabled"
-  enableReadFromArchival: true
-  defaultBucket: "cadence-development"
-  filestore:
-    storeDirectory: "/tmp/development/blobstore/"
-    defaultBucket: "cadence-development"
-    customBuckets:
-      - "custom-bucket-1"
-      - "custom-bucket-2"
-#  s3store:
-#    region: "us-east-1"
-#    endpoint: "http://127.0.0.1:4572"
-#    s3ForcePathStyle: true
+  history:
+    status: "disabled"
+    enableReadFromArchival: false
+    archiverProvider:
+      filestore:
+        fileMode: "0600"
+        dirMode: "0700"
+      # s3store:
```

Comment:
Lets not include this S3 for the time being. I suspect its going to look different than this now that we are using archiver not blobstore. I don't want this to be misleading.